src/decode.h

```diff
@@ -238,34 +238,56 @@ typedef uint16_t Port;
 #define PKT_IS_TOSERVER(p)  (((p)->flowflags & FLOW_PKT_TOSERVER))
 #define PKT_IS_TOCLIENT(p)  (((p)->flowflags & FLOW_PKT_TOCLIENT))
 
+typedef enum PacketAlertFlags {
+    /** flag to indicate the rule action (drop/pass) needs to be
+     * applied to the flow */
+    PACKET_ALERT_FLAG_APPLY_ACTION_TO_FLOW = BIT_U8(0),
```

Comment:
Putting this in an enum makes for nice documentation, are there reasons not to put flags in enum's like this?

Reply:
Types are off. I believe enum is int or unsigned int, but we use as u8.

What I've seen in other code (kernel?) is:
```c
enum {
   VALUE = 0,
#define VALUE_FLAG BIT_U8(VALUE)
   VALUEX = 1,
#define VALUEX_FLAG BIT_U8(VALUEX)
}
```

Reply:
Will look into that, or just go back to `define` flags and see if I can group the documentation a little.